index.js

```diff
@@ -689,18 +694,31 @@ float roughnessFactor = roughness;
         /* if (!meshData) {
           debugger;
         } */
+        const chunkSize = chunkWorldSize * chunk.lod;
+
         localSphere.center.set(
-            (chunk.min.x + 0.5) * chunkWorldSize,
-            (chunk.min.y + 0.5) * chunkWorldSize,
-            (chunk.min.z + 0.5) * chunkWorldSize
+            (chunk.min.x + 0.5) * chunkSize,
+            (chunk.min.y + 0.5) * chunkSize,
+            (chunk.min.z + 0.5) * chunkSize
           )
           .applyMatrix4(this.matrixWorld);
         localSphere.radius = chunkRadius;
+
+        localVector3D.set(chunk.min.x, chunk.min.y, chunk.min.z).multiplyScalar(chunkSize); // min

```

Comment:
I would give these variables names to make the below function arguments clearer.